volatility3/framework/layers/qemu.py

```diff
@@ -62,8 +62,10 @@ def _read_configuration(self, base_layer: interfaces.layers.DataLayerInterface,
                 if b'\x00' in data:
                     start = data.rfind(b'\x00')
                     data = data[data.find(b'{', start):]
-                    return json.loads(data)
-        raise exceptions.LayerException(name, "Could not load JSON configuration from the end of the file")
+                    if len(data) > 1:
```

Comment:
I'd kinda prefer the check were whether the find was `>= 0`, since that requires less comprehension to figure out that data[-1:] will still have a size of 1.  Otherwise, looks good.  5:)